Были написаны маппинги #5

eovelgan opened the pull request.

eovelgan committed:
Были написаны маппинги #5

MarinaPaley commented on Dentistry/DataAccessLayer/ORM/AppointmentMap.cs:
```diff
@@ -0,0 +1,36 @@
+            this.Map(x => x.Client);
+
+            this.References(x => x.Service)
+                .Cascade.Delete();
```
на стороне многие

MarinaPaley commented on Dentistry/DataAccessLayer/ORM/AppointmentMap.cs:
```diff
@@ -0,0 +1,36 @@
+            this.References(x => x.Service)
+                .Cascade.Delete();
+            this.References(x => x.Doctor)
+                .Cascade.Delete();
```
на стороне многие

MarinaPaley commented on Dentistry/DataAccessLayer/ORM/AppointmentMap.cs:
```diff
@@ -0,0 +1,36 @@
+            this.References(x => x.Doctor)
+                .Cascade.Delete();
+            this.References(x => x.Client)
+                .Cascade.Delete();
```
на стороне многие

MarinaPaley commented on Dentistry/DataAccessLayer/ORM/SpecializingMap.cs:
```diff
@@ -0,0 +1,26 @@
+            this.Id(x => x.IdSpecializing).GeneratedBy.Guid();
+
+            this.Map(x => x.Title).Length(255);
+
```
`this.HasMany( x => x.Doctors)
.Not.Inverse()
.Cascade.Delete();`

eovelgan committed:
Исправлены ошибки и сделаны тесты #6

1. Дописаны маппинги
2. Написаны тесты на маппинги
3. Написаны репозитории
4. Написаны тесты на репозитории
